src/main/java/jenkins/plugins/openstack/compute/internal/Openstack.java

```diff
@@ -92,12 +92,24 @@ public Openstack(@Nonnull String endPointUrl, @Nonnull String identity, @Nonnull
         String[] id = identity.split(":", 2);
         String tenant = id.length > 0 ? id[0] : "";
         String username = id.length > 1 ? id[1] : "";
-        client = OSFactory.builder().endpoint(endPointUrl)
-                .credentials(username, credential.getPlainText())
-                .tenantName(tenant)
-                .authenticate()
-                .useRegion(region)
-        ;
+        if (Jenkins.getInstance() != null && Jenkins.getInstance().proxy != null){
+            ProxyConfiguration proxy = Jenkins.getInstance().proxy;
+            LOGGER.info("So we have a proxy defined with the following details:");
+            LOGGER.info("proxy.name= " +proxy.name);
+            LOGGER.info("proxy.port= " +proxy.port);
+            client = OSFactory.builder().endpoint(endPointUrl).credentials(username, credential.getPlainText())
+                    .tenantName(tenant)
+                    .withConfig(Config.newConfig().withSSLVerificationDisabled().withProxy(ProxyHost.of(HTTP_PREFIX +proxy.name,proxy.port,proxy.getUserName(), proxy.getPassword())))
```

Comment:
There is no need to replicate the whole call chain because of a single argument changed.